frontend/src/styles/_reactselect.scss

```diff
@@ -0,0 +1,27 @@
+.react-select {
+  // Focus state
+  &__control {
+    &--is-focused {
+      border-color: #8099b3 !important;
+      box-shadow: 0 0 0 0.25rem rgba(1, 51, 102, 0.25) !important;
```

Comment:
`#8099b3` is used in https://digitalspace.github.io/bcparks-bootstrap-theme/ but not in the BC Gov design token.